sources/T-TUT-CABLETV-2020.adoc

```diff
@@ -0,0 +1,253 @@
+= Integrated broadcast-broadband digital TV application cooperated with server for functional extension including functions of digital TV reception and processing
+:bureau: T
+:docnumber: JSTP-IBBDTV
+:title-en: Integrated broadcast-broadband digital TV application cooperated with server for functional extension including functions of digital TV reception and processing
+:published-date: 2020-04
+:copyright-year: 2020
+:status: in-force
+:language: en
+:doctype: technical-paper
+:keywords: IBB system, server-side processing, STB, streaming
```

Comment:
fullname_1 etc are not defined, the code currently only recognises fullname or fullname_2. Same applies to all other attributes and docs

Reply:
Corrections are applied.